tests/e2e/specs/modules/analytics/setup-with-account-no-tag.test.js

```diff
@@ -157,6 +158,7 @@ describe( 'setting up the Analytics module with an existing account and no exist
 			// See the selects populate
 			await expect( page ).toMatchElement( '.mdc-select__selected-text', { text: /test account a/i } );
 			await expect( page ).toMatchElement( '.mdc-select__selected-text', { text: /test property x/i } );
+			// NOTE - according to spec this should be hidden. Same in storybook. Displays when account is selected. TO INVESTIGATE
 			await expect( page ).toMatchElement( '.mdc-select__selected-text', { text: /test profile x/i } );
```

Comment:
Should this profile select be hidden? I assumed so (and then I would change this line as I did with line 150)

Is the account Id and profile Id being set automatically when the account is selected?

Reply:
See here the interactive demo in storybook https://google.github.io/site-kit-wp/storybook/pull/3330/?path=/story/analytics-module--account-property-profile-select